### PR TITLE
Update fabla.rst

### PR DESCRIPTION
**Pull request by trebmuh**
two typos

### CODE REVIEW

File: source/fabla.rst [hunk 1]
```diff
@@ -47,7 +47,7 @@ ADSR
 ----
 
 The ADSR available in the top right of the GUI allows shaping of each pads
-envelope. When a pad is clicked on, the ADSR shows the currently highliged
+envelope. When a pad is clicked on, the ADSR shows the currently highligthed
```
Comment (harryhaaren) [inline source/fabla.rst]: There's a typo in your typo fix xD

Comment (trebmuh) [reply source/fabla.rst]: ahahah :)
fixed now (hopefully)

Comment (trebmuh) [reply source/fabla.rst]: fixed

**Comment by harryhaaren** [comment]
Thanks!

**Comment by trebmuh** [comment]
:+1:

**Pull request merged**